alacritty_terminal/src/config/scrolling.rs

```diff
@@ -12,7 +12,8 @@ pub struct Scrolling {
     #[serde(deserialize_with = "failure_default")]
     multiplier: ScrollingMultiplier,
     #[serde(deserialize_with = "failure_default")]
-    faux_multiplier: ScrollingMultiplier,
+    // TODO: DEPRECATED
+    faux_multiplier: Option<ScrollingMultiplier>,
```

Comment:
Usually we move deprecated options to the bottom of the structs to make it easier to find them. You should be able to see this in other places where items have been deprecated too.

Also a newline between the deprecated item and the last non-deprecated will make it stand out even more.